contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java

```diff
@@ -162,35 +160,75 @@ public void activate(Context.OperatorContext context)
       }
       getters.add(getter);
     }
+    super.activate(context);
+  }
+
+  private void populateFieldInfosFromPojo(ColumnDefinitions rsMetaData)
+  {
+    fieldInfos = Lists.newArrayList();
+    Field[] fields = pojoClass.getDeclaredFields();
+    for (int i = 0; i < rsMetaData.size(); i++) {
+      String columnName = rsMetaData.getName(i);
+      String pojoField = getMatchingField(fields, columnName);
+      if (pojoField != null && pojoField.length() != 0) {
+        fieldInfos.add(new FieldInfo(columnName, pojoField, null));
+      } else {
+        LOG.error("Couldn't find corrosponding pojo field for column: " + columnName);
```

Comment:
I haven't tried it yet, but if not null constraint is defined it should throw exception. Ideally the app developer should populate the fields as per the table schema.
We can do a quick schema check and throw exception before hand but I wonder will that do much of value addition?